libsel4/sel4_plat_include/tqma8xqp1gb/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021, Breakaway Consulting Pty. Ltd.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <autoconf.h>
+
+/* Cortex A35 Manual Table C7.1 */
+#define seL4_NumHWBreakpoints (6)
+#define seL4_NumExclusiveBreakpoints (6)
+#define seL4_NumExclusiveWatchpoints (4)
+#ifdef CONFIG_HARDWARE_DEBUG_API
+#define seL4_FirstWatchpoint (6)
+#define seL4_NumDualFunctionMonitors (0)
+#endif
+
+#if CONFIG_WORD_SIZE == 32
```

Comment:
Could you add a comment there why this is unsupported? Seems Cortex-A35 supports AARCH32, so it this just a guard for untested terrain. Maybe blocking `CONFIG_ARCH_AARCH32` builds here is more intuitive then.

Reply:
I don't really care, but CONFIG_WORD_SIZE == 32 is used in the other constants.h files, so I think it makes sense to be consistent.

Yes, the Cortex-A35 supports AARCH32, but the platform support provided for the tqma8xqp1gb is only for running in 32-bit. I'd be more than happy for someone else to get it running in 32-bit mode, but that is beyond what I can support.

This is really just a backstop as there is no configuration option to enabled 32-bit mode. Other constants.h file do conditional tests based on CONFIG_WORD_SIZE so it seems appropriate and consistent with existing code.

Reply:
I'd recommend to make this really specific here and clearly say what's going on. What the other platforms are doing seems a bit sloppy also. And none really blocks AARCH32, the just restrict the address space. 

```suggestion
#if CONFIG_ARCH_AARCH32
/* Platform support for tqma8xqp1gb is provided for AARCH64 only, even if the
 * Cortex-A35 supports AARCH32 also. Keep this as a build blocker as long as
 * AARCH32 remains untested.
 */
#error "AARCH32 is unsupported"
#endif /* CONFIG_ARCH_AARCH32 */

```

Reply:
Fixed.